main.go

```diff
@@ -208,7 +227,20 @@ func setupReconcilersHub(mgr ctrl.Manager) {
 }
 
 func main() {
-	mgr, ramenConfig, err := newManager()
+	logOpts := configureLogOptions()
+	bindFlags(logOpts.BindFlags)
+	flag.Parse()
+
+	ctrl.SetLogger(zap.New(zap.UseFlagOptions(logOpts)))
+
+	ctrlOptions, ramenConfig := buildOptions()
+
+	if err := configureController(ramenConfig); err != nil {
+		setupLog.Error(err, "unable to configure controller")
+		os.Exit(1)
+	}
+
+	mgr, err := newManager(ctrlOptions)
```

Comment:
We can possibly drop the `newManager` function and invoke `ctrl.NewManager` directly here.

Reply:
Ack no worries.